doc/how-to-run-in-windows.rst

```diff
@@ -0,0 +1,142 @@
+How to use Cabal in Windows
+===========================
+
+This document describes how to use Cabal in a Windows system. See the
+:ref:`Further reading` section for some other references that might provide some
+more explanations. For a TL;DR, jump to the :ref:`Complete configuration`.
+
+Install the Haskell environment
+-------------------------------
+
+The recommended way of setting up a Haskell environment in Windows is by using
+`GHCup <https://www.haskell.org/ghcup/>`_. Follow the steps outlined in its
+webpage to install at least GHC and Cabal. GHCup will install its own MSYS2
+system in your computer unless told not to do so, refer to `its documentation
+<https://www.haskell.org/ghcup/install/#windows_1>`_ for more information.
+
+Ensure that Cabal can call UNIX-like tools

```

Comment:
Is it accurate to categorise the tools as 'UNIX-like'? Most don't come with Windows 'out of the box', but I don't think that is unique to Windows. I would put it like this: 'Ensure that Cabal can call the tools it needs' and 'Cabal sometimes needs to call tools that do not come with Windows. The MSYS2 project makes many of them available on Windows.'

Reply:
Good point, rectified. Thanks!